label_studio/projects/functions/next_task.py

```diff
@@ -141,9 +143,30 @@ def get_not_solved_tasks_qs(user, project, prepared_tasks, assigned_flag, queue_
         user_postponed_tasks = postponed_drafts.distinct().values_list('task__pk', flat=True)
         not_solved_tasks = not_solved_tasks.exclude(pk__in=user_postponed_tasks)
 
-    # if annotator is assigned for tasks, he must to solve it regardless of is_labeled=True
+    # if annotator is assigned for tasks, he must solve it regardless of is_labeled=True
     if not assigned_flag:
-        not_solved_tasks = not_solved_tasks.filter(is_labeled=False)
+        # include tasks that have been completed if their agreement is not at threshold if threshold setting is set
+        lse_project = getattr(project, 'lse_project', None)
+        if (
+            lse_project
+            and lse_project.agreement_threshold is not None
+            and get_tasks_agreement_queryset
+            and user.is_annotator
```

Comment:
Only annotator roles are considered for low agreement tasks?

Reply:
Correct, this was something I checked with Steph on